### PR TITLE
DTDL v2 spec, keeps v1 spec, description text in deep link

### PR DESCRIPTION
**Pull request by fsautomata**
This PR adds v2 specs, and keeps v1.
It also keeps the deeplink to DTDL/README.md working and pointing to the right spec.

### CODE REVIEW

File: DTDL/v2/dtdlv2.md [hunk 1]
```diff
@@ -0,0 +1,874 @@
+# Digital Twins Definition Language (DTDL)
```
Comment (briancr-ms) [inline DTDL/v2/dtdlv2.md:1]: This isn't the latest version of the DTDL v2 markdown file. Please get it from this PR: [https://github.com/Azure/IoTPlugandPlay/pull/55](https://github.com/Azure/IoTPlugandPlay/pull/55).

Comment (fsautomata) [reply DTDL/v2/dtdlv2.md:1]: I updated from the PR. Can you verify it got it right?

File: DTDL/README.md [hunk 2]
```diff
@@ -1,632 +1,17 @@
...
-[Property](#property)<br>
-[Command](#command)<br>
-[Primitive Schemas](#primitive-schemas)<br>
-[Array](#array)<br>
-[Enum](#enum)<br>
-[Map](#map)<br>
-[Object](#object)<br>
-[Interface Schemas](#interface-schemas)<br>
-[Model Versioning](#model-versioning)<br>
+The Digital Twin Definition Language (DTDL) is a language for describing models and interfaces for IoT digital twins. Digital twins are models of entities in the physical environment such as shipping containers, rooms, factory floors, or logical entities that participate in IoT solutions. Using DTDL to describe a digital twin's capabilities enables the IoT platform and IoT solutions to leverage the semantics of the entity.
```
Comment (briancr-ms) [inline DTDL/README.md:3]: >Twin [](start = 12, length = 4)

Should be plural (Twins).

File: DTDL/README.md [hunk 3]
```diff
@@ -1,632 +1,17 @@
...
 
-## Introduction
-This document specifies the Digital Twin Definition Language (DTDL), a language for describing models and interfaces for IoT digital twins. Digital twins are models of entities in the physical environment such as shipping containers, rooms, factory floors, and other entities that participate in IoT solutions. Broadly, modeling enables IoT solutions to provision, use, and configure IoT devices and logical entities from multiple sources in a single solution. Using DTDL to describe a digital twin's capabilities enables the IoT platform and IoT solutions to leverage the semantics of the entity.
+DTDL is open to the community and Microsoft welcomes collaboration with customers, partners, and the industry. It is based on open W3C standards such as JSON-LD and RDF which allow for easier adoption across services and tooling.
 
-## Digital Twin Definition Language
+To converge the use of DTDL across Azure IoT Hub, Azure IoT Central, Azure Digital Twins, and other Azure services, DTDL is going to a set of breaking changes.
```
Comment (briancr-ms) [inline DTDL/README.md:7]: >DTDL is going to a set of breaking changes [](start = 116, length = 42)

This phrase, "DTDL is going to a set of breaking change", doesn't make sense to me. What do you mean?

**Pull request merged**